neo/rawio/intanrawio.py

```diff
@@ -53,40 +56,90 @@ class IntanRawIO(BaseRawIO):
        name of the 'rhd' or 'rhs' data file
     """
 
-    extensions = ["rhd", "rhs"]
+    extensions = ["rhd", "rhs", "dat"]
     rawmode = "one-file"
 
     def __init__(self, filename=""):
-        BaseRawIO.__init__(self)
 
+        BaseRawIO.__init__(self)
         self.filename = filename
 
     def _source_name(self):
         return self.filename
 
     def _parse_header(self):
 
+        filename = Path(self.filename)
+        
+        if not filename.exists() or not filename.is_file():
+            raise FileNotFoundError(f"{filename} does not exist")
+
         if self.filename.endswith(".rhs"):
+            self.file_type = 'header-attached'
```

Comment:
What do you think of calling this format instead of file type?

Reply:
Done.